wavefront/server/modules/gold_module/gold_module/models/gold_image_request.py

```
@@ -34,9 +34,9 @@ class ImageMetadata(BaseModel):
     gold_loan_category: str = None
     loan_tenure: int = None
     loan_amount: float = None
-    loan_type: Literal['new', 'top_up'] = None
-    pos: float = None
-    parent_loan_id: str = None
+    loan_type: Optional[Literal['new', 'top_up']] = None
```

Ruff (0.15.20)

[error] Undefined name `Optional`

(F821)